event-management-frontend/src/reducers/HeaderEventCrudReducer.tsx

```diff
@@ -15,36 +15,87 @@ import {
   UPDATE_FORM_ERRORS,
   UPDATE_EVENT,
   UPDATE_LOCATION,
-  RESET_STORE
-} from "../actions/HeaderEventCrudActions"
-import { EventCrud } from "../model/EventCrud"
-import { EventImage } from "../model/EventImage"
-import { EventFormErrors } from "../model/EventFormErrors"
+  RESET_STORE,
+  ADD_EMPTY_CATEGORY_CARD,
+  REMOVE_CATEGORY_CARD,
+} from "../actions/HeaderEventCrudActions";
+import { EventCrud } from "../model/EventCrud";
+import { EventImage } from "../model/EventImage";
+import { EventFormErrors, CategoryCardErrors } from "../model/EventFormErrors";
+import { CategoryCardItem } from "../types/TicketType";
+import { TicketAvailabilityData } from "../model/TicketAvailabilityData";
 
 export interface EventState {
-  loading: boolean,
-  event: EventCrud,
-  error: string,
-  isError: boolean,
-  isLoading: boolean,
-  images: EventImage[],
-  formErrors: EventFormErrors,
+  loading: boolean;
+  event: EventCrud;
+  ticketData: TicketAvailabilityData[];
+  error: string;
+  isError: boolean;
+  isLoading: boolean;
+  images: EventImage[];
+  formErrors: EventFormErrors;
 }
 
-let today = new Date(new Date().toString().split('GMT')[0] + ' UTC').toISOString().split('.')[0]
+let today = new Date(new Date().toString().split("GMT")[0] + " UTC").toISOString().split(".")[0];
 const dateAndTime = today.split("T");
 const currDate = dateAndTime[0];
-const currTime = dateAndTime[1].replace(/:\d\d([ ap]|$)/, '$1');
+const currTime = dateAndTime[1].replace(/:\d\d([ ap]|$)/, "$1");
 
+export const noCardError: CategoryCardErrors = {
+  title: "",
+  subtitle: "",
+  price: "",
+  description: "",
+  ticketsPerCategory: "",
+};
+export const noCardError2: CategoryCardErrors = {
```

Comment:
Foloseste o denumire mai explicita pentru "noCardError2" (eventual, pune in denumire motivul pentru iti trebuie aceasta a doua variabila noCardError - ex: "noCardErrorForUpdate")